src/App.js

```diff
@@ -12,7 +12,7 @@ export default function App() {
 				<Routes>
 					<Route path="/">
 						<Route index element={<Home />} />
-						<Route path="about/" element={<About />} />
+						<Route path="result/" element={<Result />} />
```

Comment:
As your `Home` component is empty I recommend you to remove the navigation for now as it is not necessary fot the functionality of your feature. Implement it later when you need two or more pages. :)